README.md

```diff
@@ -334,7 +335,8 @@ database for error resolution and debugging.
 
 ### Defining Relationships Between Subflows
 
-Subflows can be related to one another using YAML configuration. This looks like:
+There may be a need to have a subflow, iterations pre-determined by another subflow. Subflow relationships allow for 
```

Comment:
```suggestion
There may be a need to have one subflow's iterations pre-determined by another subflow. Subflow relationships allow for 
```

Reply:
I was looking for the suggestion but for some reason it didn't show up for me 😭